pkg/maps/metricsmap/metricsmap.go

```diff
@@ -27,7 +29,11 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+// Metrics is the bpf metrics map
+var Metrics *bpf.Map
 var log = logging.DefaultLogger
+var file *os.File
+var possibleCpus int
```

Comment:
you can group var declarations as follows:

```
var (
    a = b
    c =d 
)

Reply:
sure

Reply:
done